packages/arb-token-bridge-ui/src/components/common/SidePanel.tsx

```diff
@@ -24,22 +23,11 @@ export const SidePanel = ({
 }: SidePanelProps) => {
   const ANIMATION_DURATION = 300
 
-  const [isClosing, setIsClosing] = useState(false)
-  const [isAnimating, setIsAnimating] = useState(true)
-
-  const handleOnClose = () => {
-    setIsClosing(true)
-    onClose?.()
-    setTimeout(() => {
-      setIsClosing(false)
-    }, ANIMATION_DURATION) // Matches the transition duration
-  }
-
   return (
-    <Transition show={isOpen && !isClosing}>
+    <Transition show={isOpen}>
       <Dialog
-        open={isOpen || isClosing}
-        onClose={handleOnClose}
+        open={isOpen}
+        onClose={() => onClose?.()}
```

Comment:
might as well set a default value to the `onClose` param and just use `onClose` here

Reply:
Discussed offline, it's the same as in prod so we keep it as is